src/compilation/build.sh

```diff
@@ -504,17 +524,29 @@ function build_gdb() {
     local gdb_dir="$1"
     local target_arch="$2"
     local libiconv_prefix="$3"
-    local liblzma_prefix="$4"
-    local libgmp_prefix="$5"
-    local libmpfr_prefix="$6"
-    local with_python="$7"
-
-    if [[ "$with_python" == "yes" ]]; then
-        local python_flag="--with-python=/app/gdb/build/packages/cpython-static/build-$target_arch/bin/python3-config"
-        local gdb_build_dir="$(realpath "$gdb_dir/build-${target_arch}_with_python")"
+    local libgmp_prefix="$4"
+    local libmpfr_prefix="$5"
+    local liblzma_prefix="$6"
+    local full_build="$7"
+    local gdb_bfd_archs="$8"
+    local target_arch="$9"
+
+    local extra_flags=()
+    if [[ "$full_build" == "yes" ]]; then
+        if [[ $full_build_cross_arch_debugging -eq 1 ]]; then
+            extra_flags+=("--enable-targets=$gdb_bfd_archs" "--enable-64-bit-bfd" "--disable-sim")
+        fi
+
+        if [[ $full_build_python_support -eq 1 ]]; then
+            extra_flags+=("--with-python=/app/gdb/build/packages/cpython-static/build-$target_arch/bin/python3-config")
+        else
+            extra_flags+=("--without-python")
+        fi
+
+        local gdb_build_dir="$(realpath "$gdb_dir/build-${target_arch}_full")"
     else
-        local python_flag="--without-python"
-        local gdb_build_dir="$(realpath "$gdb_dir/build-${target_arch}")"
+        extra_flags+=("--without-python")
+        local gdb_build_dir="$(realpath "$gdb_dir/build-${target_arch}_slim")"
```

Comment:
```suggestion
        local gdb_build_dir="$(realpath "$gdb_dir/build-${target_arch}-slim")"
```

This is how you called the build alias, and I want them to be compatible.